### PR TITLE
Pull Request Template Suggestions

### PR DESCRIPTION
**Pull request by apopiak**
This PR adjusts the Pull Request template.

Changes:
+ fix typo
+ add space and separator above the template (so people can start typing directly without needing to remove the template)
+ adjust some wording
+ add super short bold descriptions of the bullet points
+ mention that a Cumulus companion would be nice

### CODE REVIEW

**Comment by bkchr** [comment]
I personally find that the template is way too much bloated. Personally it just makes it more complicated to open a pr :P 

Can we not come up with a smaller one that is more precise?

**Comment by apopiak** [comment]
Sure, open to suggestions. I just didn't want to deviate too much from what was there.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
   - There is no commented code checked in unless necessary.
-  - Any panickers have a proof or removed.
-- [ ] You bumped the runtime version if there are breaking changes in the **runtime**.
-- [ ] You updated any rustdocs which may have changed
-- [ ] Has the PR altered the external API or interfaces used by Polkadot? Do you have the corresponding Polkadot PR ready?
+  - Any panickers in the runtime have a proof or were removed.
+- [ ] **Runtime Version:** You bumped the runtime version if there are breaking changes in the **runtime**.
+- [ ] **Docs:** You updated any rustdocs which may need to change.
+- [ ] **Polkadot Companion:** Has the PR altered the external API or interfaces used by Polkadot?
+  - [ ] Do you have the corresponding Polkadot PR ready?
```
Comment (gilescope) [inline docs/PULL_REQUEST_TEMPLATE.md:28]: Polkadot PRs are not always necessary: 
```suggestion
  - [ ] If so, do you have the corresponding Polkadot PR ready?
```

**Review by gilescope — APPROVED** [review]

**Comment by bkchr** [comment]
> Sure, open to suggestions. I just didn't want to deviate too much from what was there.

I mean this is just a c&p of the contributors file. We could even just link to it.

**Review by thiolliere — APPROVED** [review]
looks good to me

**Comment by apopiak** [comment]
bot merge

**Review by kianenigma — APPROVED** [review]

**Pull request merged**